.travis.yml

```diff
@@ -1,20 +1,57 @@
 language: php
 
-env:
-    - BEHAT_SUITE=standalone
-    - BEHAT_SUITE=embedded
-    - BEHAT_SUITE=cli
-
-php:
-    - 5.6
-    - 7.0
-    - 7.1
-    - 7.2
+matrix:
+  include:
+    - php: 5.6
+      env:
+        - PREFER="--prefer-lowest"
+        - BEHAT_SUITE=standalone
+        - SYMFONY_VERSION=^2.8
+    - php: 5.6
+      env:
+        - PREFER="--prefer-lowest"
+        - BEHAT_SUITE=embedded
+        - SYMFONY_VERSION=^3.0
+    - php: 7.0
+      env:
+        - PREFER=""
+        - BEHAT_SUITE=embedded
+        - SYMFONY_VERSION=^2.8
+    - php: 7.0
+      env:
+        - PREFER=""
+        - BEHAT_SUITE=cli
+        - SYMFONY_VERSION=^3.0
+    - php: 7.1
+      env:
+        - PREFER=""
+        - BEHAT_SUITE=standalone
+        - SYMFONY_VERSION=^2.8
+    - php: 7.1
+      env:
+        - PREFER=""
+        - BEHAT_SUITE=embedded
+        - SYMFONY_VERSION=^4.0
+    - php: 7.2
+      env:
+        - PREFER=""
+        - BEHAT_SUITE=standalone
+        - SYMFONY_VERSION=^2.8
+    - php: 7.2
+      env:
+        - PREFER=""
+        - BEHAT_SUITE=embedded
+        - SYMFONY_VERSION=^3.0
+    - php: 7.2
+      env:
+        - PREFER=""
+        - BEHAT_SUITE=cli
+        - SYMFONY_VERSION=^4.0
 
 before_script:
     - composer self-update
-    - composer require "symfony/symfony" "~2.8" --no-update
-    - composer install
+    - composer require "symfony/symfony" "$SYMFONY_VERSION" --no-update
```

Comment:
requiring symfony/symfony will hide dependency problems. the proper solution these days would be something with flex as explained in https://github.com/symfony/lts

but this is no problem introduced with this pull request, so we don't have to fix it in this PR.